spec/abi.dd

```diff
@@ -238,11 +244,11 @@ $(H3 $(LNAME2 name_mangling, Name Mangling))
         $(GNAME TemplateArgX):
             $(B T) $(GLINK Type)
             $(B V) $(GLINK Type) $(GLINK Value)
-            $(B S) $(GLINK LName)
+            $(B S) $(GLINK SymbolName)
+            $(B S) $(GLINK MangledName)
```

Comment:
Actually, this should be `QualifiedName` / `MangledName`.

Example:  https://github.com/dlang/phobos/blob/31e1b462d2311f9d8cecb33172b28eab587b255d/std/parallelism.d#L43-L77

Instantiates this symbol:
```
std.parallelism.Task!(run, real delegate(MapResult!(getTerm, Result), ulong, ulong) @system, MapResult!(getTerm, Result), ulong, ulong).Task.impl
```

Mangled as:
```
_D3std11parallelism301__T4TaskS213std11parallelism3runTDFS3std9algorithm9iteration91__T9MapResultS31_D8parallel4mainFZ7getTermMFiZeTS3std5range13__T4iotaTiTiZ4iotaFiiZ6ResultZ9MapResultmmZeTS3std9algorithm9iteration91__T9MapResultS31_D8parallel4mainFZ7getTermMFiZeTS3std5range13__T4iotaTiTiZ4iotaFiiZ6ResultZ9MapResultTmTmZ4Task4implFPvZv
```

Right at the beginning of the first template you have:
```
__T4TaskS213std11parallelism3run
           ^
```
Also, this cunningly forgets Bugzilla 3043 where symbol arguments are prefixed with their length.

The correct grammar should be:
```
         $(GNAME TemplateArgX):
             $(B T) $(GLINK Type)
             $(B V) $(GLINK Type) $(GLINK Value)
            $(B S) $(GLINK LName)
            $(B S) $(GLINK Number) $(GLINK QualifiedName)
            $(B S) $(GLINK Number) $(GLINK MangledName)
```